packages/component_code_gen/templates/generate_apps.py

```diff
@@ -0,0 +1,466 @@
+
+The app can also be an OAuth app. For OAuth integrations, this object exposes the OAuth access token in the variable `this.$auth.oauth_access_token`. When you make the API request, make sure to use the format from the app docs, e.g. you may need to pass the OAuth access token as a Bearer token in the Authorization header.
+
+The app code should contain a `propDefinitions` property, which are the definitions for the props. Props lets the user pass data to the step via a form in the Pipedream UI, so they can fill in the values of the variables. Include any required parameters as properties of the `props` object. Props must include a human-readable `label` and a `type` (one of string|boolean|integer|object) that corresponds to the Node.js type of the required param. string, boolean, and integer props allow for arrays of input, and the array types are "string[]", "boolean[]", and "integer[]" respectively. Complex props (like arrays of objects) can be passed as string[] props, and each item of the array can be parsed as JSON. If the user asks you to provide an array of object, ALWAYS provide a `type` of string[]. Optionally, props can have a human-readable `description` describing the param. Optional parameters that correspond to the test code should be declared with `optional: true`. Recall that props may contain an `options` method.
+
+The `methods` property contains auxiliary methods. A `async _makeRequest` method is always required. It contains the code that makes the API request. It takes two arguments: `$ = this` and `opts`. `$` is the context passed by the Pipedream runtime. It should default to `this`. `opts` is an object that contains the parameters of the API request. The `opts` object may contain the following fields: `method`, `path`, `data`, `params`, and `headers`. The `method` field is the HTTP method of the request. The `path` field is the path of the request. The `data` field is the body of the request. The `params` field is the query parameters of the request. The `headers` field is the headers of the request. The `opts` object also contains any other fields that are passed to the `_makeRequest` method. The `_makeRequest` method returns a Promise that resolves to the HTTP response data. There is NO `data` property in the response that contains the data. The data from the HTTP response is returned directly in the response, not in the `data` property.
+
+The axios request uses the authentication method defined by the app. Different apps pass credentials in different places in the HTTP request, e.g. headers, url params, etc. The axios request should use the format from the app docs.
```

Comment:
Here again I think you could write the English specific to the app so you can tell it exactly how you should pass credentials, since we should know that for every integrated app.